### PR TITLE
Add unit tests for getMembers utils

### PR DESCRIPTION
**Pull request by wacii**

### CODE REVIEW

File: src/test/utils/getMembers.test.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+describe('dependents', () => {
+  const dependents = [
+    { m_age: 18, m_role: 'member' },
+    { m_age: 17, m_role: 'member' },
+    { m_age: 19, m_disabled: true, m_role: 'member' }
+  ];
+  const notDependents = [
+    { m_age: 18, m_role: 'head' },
+    { m_age: 18, m_role: 'spouse' },
```
Comment (knod) [inline src/test/utils/getMembers.test.js]: Next iteration, might check disabled head and spouse too.

File: src/test/utils/getMembers.test.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  const notDisabled = { m_disabled: false };
+  const allMembers = [disabled, notDisabled, notDisabled];
+
+  test('isDisabled()', () => {
+    expect(isDisabled(disabled)).toBe(true);
+    expect(isDisabled(notDisabled)).toBe(false);
+  });
+
+  test('getDisabledMembers()', () => {
+    expect(getDisabledMembers(allMembers)).toHaveLength(1);
```
Comment (knod) [inline src/test/utils/getMembers.test.js]: What happens if there are no disabled members? I don't even remember what's supposed to happen.

File: src/test/utils/getMembers.test.js (hunk 1 above)
Comment (wacii) [reply src/test/utils/getMembers.test.js]: Added them.

File: src/test/utils/getMembers.test.js (hunk 2 above)
Comment (wacii) [reply src/test/utils/getMembers.test.js]: I added a test to `getEveryMember()` for this. But it's an empty array.

**Pull request merged**